frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -84,6 +90,56 @@ export default class Metadata extends React.Component<Props> {
         </Localized>;
     }
 
+    renderGroupComment(entity: Entity): React.Node {
+        if (!entity.group_comment) {
+            return null;
+        }
+
+        return <Localized id='entitydetails-Metadata--group-comment' attrs={ { title: true } }>
+            <Property title='Group Comment' className='comment'>
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } }>
+                    { entity.group_comment }
+                </Linkify>
+            </Property>
+        </Localized>;
+    }
+
+    constructor(props: Props) {
+        super(props);
+        this.state = { seeMore: false };
+    }
+
+    handleClick = () => {
+        this.setState((state) => {
+            return { seeMore: !state.seeMore };
+        });
+    };
```

Comment:
We could simply `this.setState({ seeMore: true });`, because the button is only visible when the text is truncated, i.e. when it can "see more" (and not "see less").

Reply:
Also, let's move this method up, in front of `handleClickOnPlaceable`, to keep all `render*` methods together. I'd also call it `handleClickOnSeeMore`, to follow the pattern introduced by `handleClickOnPlaceable`.